### ISSUE TITLE
GSTerr11. For your information and agreement, Country name change: Swaziland for Eswatini (Southern Africa) 

### ISSUE THREAD

**Issue by lurdesrd** [tdwg/geoschemes#12]
On April 19, 2018, King Mswati III announced that the country would cease to be known by its colonial name, officially renamed Eswatini.

Supplementary information:
https://en.wikipedia.org/wiki/Eswatini

**Comment by rg02kg** [comment]
fully agree

**Comment by RRKlopper** [comment]
Agree. The new name has been accepted in the southern African subregion.

**Comment by PacoPando** [comment]
Changes committed in working docs
